# -*- coding: utf-8 -*-
# Generated by Django 1.11.1 on 2017-10-21 17:27
from __future__ import unicode_literals

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('coriandrum', '0011_merge_20171021_1727'),
    ]

    operations = [
        migrations.AlterField(
            model_name='coriandrumuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.ASCIIUsernameValidator()], verbose_name='username'),
        ),
    ]
